docs/ssh.md

```diff
@@ -106,7 +106,10 @@ that this is a Pod-wide setting, unlike the container `securityContext` above.
 If you want git-sync to run as a different (non-root) UID and GID, you can
 change these last blocks to any UID/GID you like.  SSH demands that the current
 UID be present in /etc/passwd, so in this case you will need to add the
-`--add-user` flag to git-sync's args array. Also, you need to change `GIT_SYNC_ROOT` to some other location, say `/workspace`, instead of `$HOME/git` as the `$HOME` will be `/` for the user and, `GIT_SYNC_ROOT` will end up in `//tmp` which do not exist.
+`--add-user` flag to git-sync's args array. Also, you need to change
+`GIT_SYNC_ROOT` to some other location, say `/workspace`, instead of
+`$HOME/git` as the `$HOME` will be `/` for the user and, `GIT_SYNC_ROOT` will
+end up in `//git` which do not exist.
```

Comment:
last pick:  s/do/does

Reply:
Sorry, I did not understand this.

Reply:
sorry, - please change "do" to "does" (blame English)

Reply:
Actually, I changed it to "//git which the non-root user can't create" as suggested by you previously.

Also, I created an account in Linux Foundation ID Portal account with the correct same e-mail address but didn't get the email agreement to sign for CLA.

Reply:
@caniszczyk - how can we poke the CLA machinery to check this?

Reply:
Are you sure it was the same email you used for this commit (sh...183@gmail.com) ?

Check your spam and make sure?  We're stuck until it is resolved. :(